Beluga-Class Transport (we rich as hell!)

spockye opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
so, what is the Beluga class?
well! its a high-class transport ship, used by the rich and important. its sleek design, rich interior and high speed makes it perfect for transportation of high value passengers. 

While the ship has no science materials, barely any medical supplies, almost no security gear or engineering supplies, it DOES have a fancy kitchen, a rich interior design and a total of...      6 cryo pods!!! (rich people don't SHARE cryopods???)

it exists to fill the niche of "rich person roleplay", something that was left empty after the removal of the high class

----

the ship contains:
two bedrooms
a bunk for the crew
a large meeting room
a kitchen/bar
a inteq security checkpoint
and other essentials!

----

The crew:
1x Captain
1x Bridge officer
2x Inteq private security officers
1x bartender
1x mechanic
1x janitor
4x high class passenger

![image](https://user-images.githubusercontent.com/79304582/233655271-d4ab311a-d302-4b7a-9021-9e1998621c0f.png)

![2023 04 21-15 59 16](https://user-images.githubusercontent.com/79304582/233655325-69945d73-003c-4cba-99c3-9840e6adfb43.png)

![image](https://user-images.githubusercontent.com/79304582/233712187-633ceb13-46a7-4bd7-87d2-79148fef1a9f.png)

![image](https://user-images.githubusercontent.com/79304582/233712222-422c249e-c716-404a-8d4d-c4f6306f7ff1.png)


<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
its a unique ship that fills a currently un-used niche, and it builds onto the lore by having hired inteq guards
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
add: Adds the Beluga class transport
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

spockye committed:
main files

beluga json
beluga dmm
hop ver
law ver
sec ver

spockye committed:
final smol change

)its now roundstart(

spockye committed:
Update independent_beluga.dmm

added a sink
moved the safe
misc fixes

Erikafox reviewed (changes requested):
This needs an exterior docking port. Ping in ruinyard or look at the Borea

spockye committed:
Merge branch 'master' into high-class-2

spockye committed:
Update independent_beluga.dmm

spockye committed:
Merge branch 'high-class-2' of https://github.com/spockye/Shiptest into high-class-2

spockye committed:
adds some sidings to the airlock, expands the map by 1y

spockye committed:
adds subshuttle dock

you can now dock subshuttles with the ship

woo!

spockye committed:
switches up the books

spockye committed:
adds cameras

spockye committed:
Merge branch 'master' into high-class-2

AverageUser67 commented:
one of the bedrooms has a medical laptop the passengers cant access
the rich people have to SHARE bedrooms too

AverageUser67 commented:
also the pure fernet bottle is... Not the best choice of beverage, maybe champagne

AverageUser67 commented:
the pixelshifted rack makes items you put on it go in the default tile center
there's no hatchet to chop the wood logs
this was based on a live play test so it might be different for others idk

spockye committed:
responds to review

unpixelshifts the wood rack
(no hatchet, get creative!)
replaces drink with sake
changes security access to "1"
replaces medcomp with normal comp
thanks @averageuser67!

AverageUser67 commented:
you CAN just make a hatchet with the autolathe and roundstart materials but that's much better

spockye committed:
misc "mad dough" changes

moves the safe, and adds a lil bit more cash in it

adds a mediumrand moneyspawner in the pockets of the highclass passengers

spockye committed:
hides a special lil cloak on the shuttle

spockye committed:
Merge branch 'master' into high-class-2

spockye committed:
adds a pump for the engines

Erikafox commented on code/modules/jobs/job_types/security_officer.dm:
```diff
@@ -231,6 +231,22 @@ GLOBAL_LIST_INIT(available_depts, list(SEC_DEPT_ENGINEERING, SEC_DEPT_MEDICAL, S
 	backpack_contents = list(/obj/item/melee/baton/loaded=1, /obj/item/ammo_box/magazine/co9mm=1, /obj/item/gun_voucher=1,/obj/item/pda/security)
 
+/datum/outfit/job/security/inteq/beluga
+	name = "IRMG Enforcer (Inteq)"
```
You may want to include a (Beluga) here, as I believe the enforcer is already defined with this name elsewhere.

spockye replied:
donezo

Erikafox commented on code/modules/jobs/job_types/lawyer.dm:
```diff
@@ -37,3 +37,13 @@
+	neck = null
+	glasses = /obj/item/clothing/glasses/sunglasses/big
+	l_hand = null
+	r_pocket = /obj/item/spacecash/bundle/mediumrand
```
Maybe give them a little more.

Erikafox reviewed (changes requested):
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/1fe3b3df-e805-47c0-b1e9-f8f89e655498)
Please check to see if the sepia tiles still have slowdown, and consider giving the ship some actual planks to fuel the fireplace
![StrongDMM_E6WgeBIxi5](https://github.com/shiptest-ss13/Shiptest/assets/94164348/c344c762-fd25-4926-ab3c-5488bc9b67ad)
move this down 4 pixels please
![Uploading StrongDMM_LCXPOLAefV.png…]()
Please don't use factional secret documents here, If you'd like them to have something unique, I'd subtype something like... "Trade Secrets". Give it that macguffin value.
![StrongDMM_WygPbdVyY7](https://github.com/shiptest-ss13/Shiptest/assets/94164348/ccb4d7d1-a6d2-421b-842e-d3b854d8b5a5)
please give these windows their own button inside the high passage room. They're paying customers, they should be able to look outside
![StrongDMM_fXOwBUSsHJ](https://github.com/shiptest-ss13/Shiptest/assets/94164348/d0f4ca5b-e723-47bc-8169-718a1c75b808)
put a req access on this button so only passengers + the captain can open up the private room from outside.
![StrongDMM_nZX5G27oRS](https://github.com/shiptest-ss13/Shiptest/assets/94164348/bbd014f1-fe42-4100-9d43-3583a860b47e)
Experiment with directions here
![StrongDMM_CXXvWgcMaX](https://github.com/shiptest-ss13/Shiptest/assets/94164348/408be8c9-5bf4-4d3b-80e9-163b4dad894d)
Random syringes please, once someone figures out that there's roundstart meth. Well.
![Uploading StrongDMM_X39spB2191.png…]()
No stacking wallmounts like that
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/f6df9890-6480-4d61-9014-e98819f45133)
Consider putting a table wideband up here.
![StrongDMM_6k2d0r757p](https://github.com/shiptest-ss13/Shiptest/assets/94164348/8e1c2fcf-a8d8-4603-b608-956a3219bf31)
The curved window must go...

The ship needs newscasters. Like. Several
![StrongDMM_3SpcrgIdWZ](https://github.com/shiptest-ss13/Shiptest/assets/94164348/edb9bd94-65ea-45cd-8aa3-153bb5111a11)
random syringes here too
![StrongDMM_MZ0JVW44JF](https://github.com/shiptest-ss13/Shiptest/assets/94164348/f518579a-b19a-4b37-8ac7-c22d9e219655)
consider windows + shutters on the external part of this airlock. It might look nice.
![StrongDMM_oQ4vZ3D4D9](https://github.com/shiptest-ss13/Shiptest/assets/94164348/7b0d01da-f9cc-4bed-89f8-4b24e975cd0d)
width 30 dwidth 15
![StrongDMM_Omu3ty3w7F](https://github.com/shiptest-ss13/Shiptest/assets/94164348/367daa34-07da-48f9-96cb-a8a7a6e3a172)
I'd remove the food processor and give them a cutting board. Not only is it a ghetto alternative, but it adds some roleplay potential
![StrongDMM_nlO7lPpVeI](https://github.com/shiptest-ss13/Shiptest/assets/94164348/4190a434-af69-49b0-853b-2e1424e88c7b)
No catwalk floors.
![StrongDMM_QA6GwmMexE](https://github.com/shiptest-ss13/Shiptest/assets/94164348/f8003815-8c93-4399-9874-fbc621423af6)
The dock and the subshuttle dock should not overlap. I'd put the dock down in the cargo area.
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/b17cc918-ce22-459e-8da8-79e939b4e354)
I'd let security toggle this themselves
![StrongDMM_g2HPsW3K2l](https://github.com/shiptest-ss13/Shiptest/assets/94164348/6e2bebbd-57cd-44ba-88d0-8c9d8be6d74d)
subtype like. an inteq megaphone or something
![StrongDMM_xIKXHgIVcO](https://github.com/shiptest-ss13/Shiptest/assets/94164348/ac2c3c9a-95c3-457c-adcd-eeaba3ff6a9e)
Real Stun Batons. Telebatons have instant knockdown, stun batons have a delayed one. 
![StrongDMM_lmg0epWBxS](https://github.com/shiptest-ss13/Shiptest/assets/94164348/6486965d-a5a9-4ced-bf24-4325e4573dbc)
If this is supposed to be a dressing area, give it a door on the south

Sorry this took so long

spockye committed:
replaces (inteq) with (Beluga)

spockye committed:
Merge branch 'master' into high-class-2

Latentish reviewed (approved):
This is a good ass ship I've seen it in round, it works, the ship is nice, Approval from me.

Make sure to address Blessed's review.

spockye committed:
Merge branch 'master' into high-class-2

Signed-off-by: spockye <79304582+spockye@users.noreply.github.com>

spockye committed:
responds to HALF the review

(doing the rest later)

spockye committed:
Merge branch 'high-class-2' of https://github.com/spockye/Shiptest into high-class-2

spockye committed:
responds to the rest of the review

(still need to subtype the megaphone)

spockye committed:
dont look in here

hear ye hear ye!!
slurpcoins for all!
you win

spockye committed:
Merge branch 'master' into high-class-2

spockye committed:
Merge branch 'master' into high-class-2

spockye committed:
Merge remote-tracking branch 'upstream/master' into high-class-2

spockye committed:
Merge remote-tracking branch 'upstream/master' into high-class-2

Erikafox committed:
Update independent_beluga.dmm

Erikafox committed:
Merge branch 'high-class-2' of https://github.com/spockye/Shiptest into pr/1920
